duckduckgo/macos-browser
# New configuration for exporting builds for feedback/review

Task/Issue URL: TBD


**Description**:
I've added a special configuration "Product Review Release". It has a different icon and bundle id to prevent data collision.

**Steps to test this PR**:
1. Run "Product Review Release" and make sure it doesn't collide with the debug or release configurations


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
